src/library.js

```diff
@@ -2358,7 +2358,7 @@ mergeInto(LibraryManager.library, {
                                "  };\n" +
                                "} else " +
 #endif
-#if USE_PTHREADS
+#if USE_PTHREADS && !AUDIO_WORKLET
```

Comment:
Does this mean that just enabling `AUDIO_WORKLET` affects code not running in the worklet?

Reply:
When building with AUDIO_WORKLET enabled, we cannot unconditionally use performance.timeOrigin + performance.now(), since those don't exist in AudioWorklets.

In that case, we must use one of the forms below that feature check for presence of performance.now(), and only use that if available.

Line 2376 below will still implement the same behavior for non-Audio Worklet threads, so they get performance.timeOrigin + performance.now().